### ISSUE TITLE
[Suggestion] Delete history by visit.

### ISSUE THREAD

**Issue by happysurf** [Rayquaza01/HistoryCleaner#3]
In my opinion the best way for keep the history database cleaned and complete is delete all web sites with only 1 visit (or more).
For that will be great have also the option > **Delete History by visit** into a date period.
Thanks.

**Comment by herrbrixter** [comment]
I don't agree. I don't really care how many times I visited a website. If I visited a website once, it will get deleted anyway by this addon if it gets too old.

**Comment by Rayquaza01** [comment]
@herrbrixter Assuming you're going to go through all of the issues, I'll save you some time. This, #25 and #32 are all (I believe) are impossible for the same reason. I've been meaning to write up an explanation of why, but I haven't got around it it. The short version is that there is no API to delete a specific visit of a site, you have to delete *every* visit. 

History Cleaner uses the deleteRange API, which lets you delete visits in a specific date range, but doesn't allow for finer filtering.

**Comment by herrbrixter** [comment]
@Rayquaza01 That's fine. I don't need a blacklist or whitelist option. I think they're just nice to have features.
